docs/integrations.md

```diff
@@ -156,3 +156,25 @@ In addition, ruff publishes the following images:
 
 As with the distroless image, each image is published with ruff version tags as
 `ruff:{major}.{minor}.{patch}-{base}` and `ruff:{major}.{minor}-{base}`, e.g., `ruff:0.6.6-alpine`.
+
+## GitLab CI/CD
```

Comment:
Should we move this right after GitHub Actions to have all CI in one place?

Reply:
If you prefer to, sure